### ISSUE TITLE
Cabelos

### ISSUE THREAD

**Issue by vish-blog** [vish-blog/comments#151]
Post de referência: https://www.vish.blog.br/imagens/cabelos/

**Comment by one-eyed-willy** [comment]
Se uma mulher disse, então deve ser verdade

**Comment by brunoaugusto** [comment]
> [angellisy](</perfil/?user=angellisy>) disse [terça-feira, 17 de maio de 2022](<https://www.vish.blog.br/imagens/cabelos/#comment_1129042205>):

> E como você pode ter certeza de que sou mulher?

Verdade. Ela pode ser um cara de 130kg que vive no porão da mãe no subúrbio de Detroit chamado Chuck :stuck\_out\_tongue:
